src/components/cards/CardItem.vue

```diff
@@ -37,9 +37,25 @@
 					<ActionButton icon="icon-archive" @click="archiveUnarchiveCard()">{{ t('deck', (showArchived ? 'Unarchive card' : 'Archive card')) }}</ActionButton>
 					<ActionButton v-if="showArchived === false" icon="icon-delete" @click="deleteCard()">{{ t('deck', 'Delete card') }}</ActionButton>
 					<ActionButton icon="icon-settings-dark" @click="openCard">{{ t('deck', 'Card details') }}</ActionButton>
+					<ActionButton icon="icon-settings-dark" @click.stop="modalShow=true">{{ t('deck', 'Move card') }}</ActionButton>
 				</Actions>
 
 			</transition>
+			<modal v-if="modalShow" title="Move card to another board" @close="modalShow=false">
+				<div class="modal__content">
+					<Multiselect v-model="selectedBoard" :options="boards" label="title"
+						@select="loadStacksFromBoard" />
+					<Multiselect v-model="selectedStack" :options="stacksFromBoard" label="title" />
```

Comment:
Please add proper labels to the multiselects, otherwise users don't know what they actually need to do.